frontend/src/Project/EditControls/SegmentControls/ToolButtons/SelectButton.js

```diff
@@ -9,7 +9,11 @@ function SelectButton(props) {
 
   const onClick = useCallback(() => segment.send({ type: 'SET_TOOL', tool: 'select' }), [segment]);
 
-  const tooltipText = <span>Click to select a label</span>;
+  const tooltipText = (
+    <span>
+      Click to select a label <kbd>V</kbd>
+    </span>
+  );
```

Comment:
How is the `V` key bind different from hitting esc to default to select mode?

Reply:
I believe the purpose of hitting `esc` is to "exit" from the current selection (eg. if you have cell 10 selected, you will reset to "cell 0" meaning nothing is selected), and the currently used tool will also persist. The purpose of hitting `v` is to actually change the current tool to select mode (but the current selection is still maintained). 

In fact, because hitting `esc` does not change the tool, I think I discovered a bug where hitting `esc` while using the brush tool allows you to brush cells with value 0, which I'll raise an issue for.